src/js/models/v2/awards/additionalDetails/BaseContractAdditionalDetails.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * BaseContractAdditionalDetails.js
+ * Created by Lizzie Salita 3/5/18
+ */
+
+const BaseContractAdditionalDetails = {
+    populate(data) {
+        this.pricingCode = data.type_of_contract_pricing || '';
+        this.pricing = data.type_of_contract_pric_desc || '';
+
+        // Parent Award Details
+
+        this.idvType = data.idv_type || '--';
+        this.idcType = data.type_of_idc || '--';
+        this.idvAgencyId = data.referenced_idv_agency_iden || '--';
+        this.multipleIdv = data.multiple_or_single_award_i || '--';
+
+        // Competition Details
+        this.solicitationId = data.solicitation_identifier || '--';
+        this.solicitationProcedures = data.solicitation_procedures || '--';
```

Comment:
This should use `solicitation_procedur_desc` if truthy, otherwise fall back to `solicitation_procedures` before using the `--`.

Reply:
Please review the rest of this file to ensure the correct mappings.